### PR TITLE
Profile Commands

### PR DESCRIPTION
**Pull request by spikecurtis**
Add & remove tags
Update profile rules

### CODE REVIEW

File: calicoctl.py [hunk 1]
```diff
@@ -490,12 +470,111 @@ def group_show(detailed):
...
+def profile_tag_show(name):
+    """Show the tags on the profile."""
+    try:
+        profile = client.get_profile(name)
+    except KeyError:
+        print "Profile %s not found." % name
+        sys.exit(1)
+
+    for tag in profile.tags:
+        print tag
```
Comment (tomdee) [inline calicoctl.py]: I'd prefer that we carry on using PrettyTable for consistency.

File: calicoctl.py [hunk 2]
```diff
@@ -490,12 +470,111 @@ def group_show(detailed):
...
+
+def profile_rule_show(name, human_readable=False):
+    """Show the rules on the profile."""
+    try:
+        profile = client.get_profile(name)
+    except KeyError:
+        print "Profile %s not found." % name
+        sys.exit(1)
+
+    if human_readable:
```
Comment (tomdee) [inline calicoctl.py]: Again, PrettyTable would be good here.

File: calicoctl.py [hunk 3]
```diff
@@ -490,12 +470,111 @@ def group_show(detailed):
...
+        print "Inbound rules:"
+        i = 1
+        for rule in profile.rules.inbound_rules:
```
Comment (tomdee) [inline calicoctl.py]: Use enumerate rather than i=1 and i +=1
https://docs.python.org/2/library/functions.html#enumerate

**Comment by tomdee** [comment]
LGTM - I'd prefer that we stick with prettytable for output and one minor point on using enumerate.

File: calicoctl.py (hunk 2 above)
Comment (spikecurtis) [inline calicoctl.py]: Are you ok with keeping the human readable description?  I don't want a bunch of mostly empty cells from breaking out the rule fields individually.

Comment (tomdee) [inline calicoctl.py]: Yes, that sounds sensible.

Comment (spikecurtis) [inline calicoctl.py]: I mocked this up and I must admit I don't like it.  We're not displaying tabular data (it's two lists), so readability goes down with this approach.

Comment (tomdee) [inline calicoctl.py]: OK, fair enough. I’m happy with whatever you think is sensible.

Whilst I think it’s good to be consistent, “A Foolish Consistency is the Hobgoblin of Little Minds”

**Pull request merged**